chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -568,9 +568,12 @@ def get_settings(self):
         """Translate the bit in the mem_struct into settings in the UI"""
         _mem = self._memobj
         basic = RadioSettingGroup("basic", "Basic Settings")
-        bank = RadioSettingGroup("bank", "Bank names")
         dtmfe = RadioSettingGroup("dtmfe", "DTMF Encode Settings")
-        top = RadioSettings(basic, bank, dtmfe)
+        if self._has_support_for_banknames:
+            bank = RadioSettingGroup("bank", "Bank names")
+            top = RadioSettings(basic, bank, dtmfe)
+        else:
+            top = RadioSettings(basic, dtmfe)
```

Comment:
Yeah, I understand what it's doing and why, it's just not related to the adding of the new radio.

Reply:
It's just best if they're separate commits, and separate issues of course. If we needed to revert, say, the new radio model commit, we wouldn't want to regress on the two other bug fixes just because they were all squashed together.

Anyway, I'll just create issues and split the commits.

Reply:
👍 I'll remember that for next time.